pontoon/base/templates/home.html

```diff
@@ -0,0 +1,175 @@
+              <div class="card-image"><img src="/static/img/Homepage/feature-4.svg"></img>
+              </div>
+              <div class="card-text content-style colored-content-style">Mass actions for quick translation and review tasks.
+              </div>
+            </div>
+            <div class="card-features flex vertical-align-center">
+              <div class="card-image"><img src="/static/img/Homepage/feature-5.svg"></img>
+              </div>
+              <div class="card-text content-style colored-content-style">Automated bi-di synchronization with version control systems.
+              </div>
+            </div>
+            <div class="card-features flex vertical-align-center">
+              <div class="card-image"><img src="/static/img/Homepage/feature-6.svg"></img>
+              </div>
+              <div class="card-text content-style colored-content-style">In-context localization.
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Section-5: In-context localization -->
+    <section class="snaps section-5 white-background" data-section-name="section-5">
+      <div class="container vertical-align-center container--full-height flex--wrap">
+        <div class="in-context flex-col-full flex-col-6 flex-direction-col neg-margin--large">
+          <div class="title-style colored-title-style align-left portrait-align-center">One more thing.</div>
+          <div class="content-style colored-content-style">One of the coolest features of Pontoon is localizing web content in-place, with context and spatial limitations right in front of you. The moment you submit translation, it replaces the original text in the web page, making you the first proofreader and tester.<br/></div>
+          <a class="homepage-button primary-button-cta" href="/in-context/">
+            <div>Give it a Try!</div>
+          </a>
+        </div>
+        <div  class= "flex-col-full flex-col-6 align-right portrait-align-center"><img class="section-side-image-3 neg-margin--large" src="/static/img/Homepage/section-5.svg"></img></div>
+      </div>
+    </section>
+
+    <!-- Section-6: Final -->
+    <section class="snaps section-6" data-section-name="section-6">
+      <div class="footer vertical-align-center flex-direction-col">
+        <div class="title-style">Join us.</div>
+        <p class="content-style">Now that you know what Pontoon is, come on in.</p>
+        <a class="homepage-button primary-button-cta" href="/teams/">
+          <div>Start Localizing Now</div>
+        </a>
+        <p class="content-style">Want to learn more?</p>
+        <a class="homepage-button secondary-button-cta" href="/tour/">
+          <div>Take a Tour</div>
+        </a>
+      </div>
+      <div class="homepage-footer flex">
+        <div class="flex-col-4 align-left"><a href="https://mozilla.org/"><img class="moz-logo" src="/static/img/Homepage/moz-logo.svgz"></img></a></div>
```

Comment:
Because it's the one available on the https://mozilla.design/mlogo plus svgz format is compressed svg files which supports graphic data in XML format.